.github/sync-files.yaml

```diff
@@ -7,9 +7,6 @@
     - source: .github/ISSUE_TEMPLATE/bug.yaml
     - source: .github/ISSUE_TEMPLATE/config.yml
     - source: .github/ISSUE_TEMPLATE/task.yaml
-    - source: .github/PULL_REQUEST_TEMPLATE.md
-    - source: .github/PULL_REQUEST_TEMPLATE/small-change.md
-    - source: .github/PULL_REQUEST_TEMPLATE/standard-change.md
```

Comment:
Normally these files are copied from [autoware/.github](https://github.com/autowarefoundation/autoware/tree/main/.github) folder.

To test it out in Universe for a while and to keep things simple, I've removed them from the sync list.